docs/UserGuide.md

```diff
@@ -220,16 +266,22 @@ Sorts and displays items based on sorting criteria
 
 ## Command summary
 
-| Action                | Format                                                     | Examples                                                                           |
-|-----------------------|------------------------------------------------------------|------------------------------------------------------------------------------------|
-| **list** (supplier)   | `list `                                                    | `listSuppliers `<br>`listTasks `<br>`listInventory`                                |
-| **add**  (supplier)   | `add n/NAME p/PHONE pr/PRICE i/ITEM a/ADDRESS t/Supplier`  | `add n/ABC PTE LTD p/67009000 pr/$1.10 i/Egg a/Blk 140 Woodlands Ave 3 t/Supplier` |
-| **addTask**           | `addTask d/DETAILS dl/DEADLINE t/TAGS`                     | `addTask d/Restock Eggs dl/2022-12-12 t/Urgent`                                    |
-| **delete** (supplier) | `delete <index>`                                           | `delete 1`                                                                         |
-| **deleteTask**        | `deleteTask <index>`                                       | `deleteTask 1`                                                                     |
-| **mark**              | `mark <index>`                                             | `mark 1`                                                                           |
-| **unmark**            | `unmark <index>`                                           | `unmark 2`                                                                         |
-| **edit**  (supplier)  | `edit <index of item> <attribute name>/ <new value>`       | `edit 3 pr/NEW PRICE`                                                              |
-| **editTask**          | `editTask <index of item> <attribute name>/ <new value>`   | `editTask 1 dl/2029-12-12`                                                         |
-| **find**  (supplier)  | `find <name value>`                                        | `find John Cena`                                                                   |
-| **sort**              | `sort /<item type> <sort criteria> <ascending/descending>` | `sort /task time descending`                                                       |
+| Action                       | Format                                                                | Examples                                                                           |
```

Comment:
Good that commands are separated to suppliers, task then items. This makes it easier to look for commands.